packages/concerto-vocabulary/test/org.acme@1.0.0.cto

```diff
@@ -9,7 +9,7 @@ enum Color {
 scalar SSN extends String default="000-00-0000"
 scalar VIN extends String
 
-asset Vehicle identified by vin {
+asset Vehicle identified {
```

Comment:
Why is this changed?

Reply:
No specific reasons, I was working on another feature #776 and I decided to test it that way.
I will change it back.